js/menu.js

```diff
@@ -0,0 +1,7 @@
+const menu_btn = document.querySelector('.hamburger');
+const mobile_menu = document.querySelector('.mobile-nav');
+
+menu_btn.addEventListener('click', () => {
```

Comment:
Nice 👍🏾 use of arrow functions for proper ES6 convention rules for best practices.